### ISSUE TITLE
Cordova iOS - cannot click buttons a second time if they have the loading prop

### ISSUE THREAD

**Issue by nicholaszuccarelli** [quasarframework/quasar#6957]
Got myself another wierd bug on iOS (Cordova 9, ionic webview).

If I have a button, say
`<q-btn color="green" icon-right="done" @click="saveDetails" label="Save" :loading="loading" />`

When I click on the button the first time, it activates my saveDetails method, which will set a variable `loading` to `true`.
Once the request is done, it sets `loading` back to `false`.

During this process, the loading prop on my btn will correctly show the loading spinner, however, if I try click on "Save" subsequent times afterward, it does nothing - the button cannot be clicked.

I checked the classes that Quasar adds to the button, and I noticed that the button has the class `non-selectable` (which was added when `loading = true`. I gather that it is not being removed when `loading` is reset back to `false`?

Oddly enough, this issue is only happening on Cordova iOS. If I try it in the browser this bug does not occur.

Let me know if I can provide any other details.

**Comment by nicholaszuccarelli** [comment]
EDIT:

I have just checked on Cordova android too and I am also having the issue there. Again, not on the browser.

These buttons are placed inside of a QDialog if that is of any help.

**Comment by nicholaszuccarelli** [comment]
I'm now not even sure if this is actually a problem with my dialog...

I have this button inside :

QDialog
->QCard
-->QTabPanels
--->QTabPanel
---->QToolbar
----->QBtn

This is the specific order in which it is not clickable.

**Comment by pdanpdan** [comment]
What version of quasar?

**Comment by nicholaszuccarelli** [comment]
V1.10.3

**Comment by pdanpdan** [comment]
Duplicate of #6947, #6946, #6945 
Upgrade to 1.10.4 and reopen if still present.
